build.sbt

```diff
@@ -139,13 +139,14 @@ lazy val rocketLibDeps = (rocketchip / Keys.libraryDependencies)
 // introducing a circular dependency between RC and MIDAS
 lazy val midasTargetUtils = ProjectRef(firesimDir, "targetutils")
 
-lazy val testchipip = (project in file("generators/testchipip"))
-  .dependsOn(rocketchip, sifive_blocks)
+val testchipipDir = file("generators/testchipip")
+lazy val testchipip = freshProject("testchipip", testchipipDir)
```

Comment:
Why does this need to be a fresh project now?

Reply:
I'd assume the same reason as some other projects that need to be a fresh project. It's not clear to me why but the vanilla `(project in file)` shows testchipip is not registered within sbt despite showing up in the log